changelog/fix-5231-wrong-call-to-fetch-authorization-data-from-the-transaction-details-for-failed-payments

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix wrong capture deadline date in the payment details page.
```

Comment:
Can we expand this a little bit more, so some context is present, please? This is an external communication and we should keep good standards here.

Reply:
Thanks, I hope it is better like this?

Reply:
Yes!